net-misc/x11-ssh-askpass/x11-ssh-askpass-1.2.4.1-r2.ebuild

```diff
@@ -6,26 +6,25 @@ EAPI=7
 inherit toolchain-funcs
 
 DESCRIPTION="X11-based passphrase dialog for use with OpenSSH"
-HOMEPAGE="http://www.liquidmeme.net/software/x11-ssh-askpass
-	https://github.com/sigmavirus24/x11-ssh-askpass"
-SRC_URI="http://www.liquidmeme.net/software/x11-ssh-askpass/${P}.tar.gz"
+HOMEPAGE="https://github.com/sigmavirus24/x11-ssh-askpass"
+SRC_URI="https://github.com/sigmavirus24/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
```
--2020-06-18 12:01:53--  https://codeload.github.com/sigmavirus24/x11-ssh-askpass/tar.gz/1.2.4.1
Resolving codeload.github.com... 140.82.113.9
Connecting to codeload.github.com|140.82.113.9|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [application/x-gzip]
Saving to: '/var/tmp/distfiles/x11-ssh-askpass-1.2.4.1.tar.gz.__download__'

/var/tmp/distfiles/     [  <=>               ]  29.04K   117KB/s    in 0.2s    

2020-06-18 12:01:54 (117 KB/s) - '/var/tmp/distfiles/x11-ssh-askpass-1.2.4.1.tar.gz.__download__' saved [29740]

!!! Fetched file: x11-ssh-askpass-1.2.4.1.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
!!! Got:      29740
!!! Expected: 29229
Refetching... File renamed to '/var/tmp/distfiles/x11-ssh-askpass-1.2.4.1.tar.gz._checksum_failure_.cqqpw584'

!!! Couldn't download 'x11-ssh-askpass-1.2.4.1.tar.gz'. Aborting.
 * Fetch failed for 'net-misc/x11-ssh-askpass-1.2.4.1-r2', Log file:
 *  '/var/tmp/portage/net-misc/x11-ssh-askpass-1.2.4.1-r2/temp/build.log'

>>> Failed to emerge net-misc/x11-ssh-askpass-1.2.4.1-r2, Log file:
```
:\

You probably had the distfile downloaded already or something. Or you had `GENTOO_MIRRORS` specified and it got downloaded from there. Just use `${PV}.gh.tar.gz` on this version to update mirrors.

Reply:
Thanks!